traffic_stats/traffic_stats.go

```diff
@@ -131,24 +224,24 @@ func main() {
 	tickers = setTimers(config)
 
 	termChan := make(chan os.Signal, 1)
-	signal.Notify(termChan, syscall.SIGKILL, syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT)
+	signal.Notify(termChan, syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT)
```

Comment:
Why'd you remove `SIGKILL`?

Reply:
Programs can't actually catch SIGKILL, so it was unnecessary at best, misleading at worst.